js/bytex.js

```diff
@@ -53,14 +54,12 @@ module.exports = class bytex extends Exchange {
                 'fetchFundingRateHistory': false,
                 'fetchFundingRates': false,
                 'fetchIndexOHLCV': false,
-                'fetchIsolatedPositions': false,
                 'fetchLeverage': false,
-                'fetchLeverageTiers': false,
-                'fetchMarketLeverageTiers': false,
```

Comment:
`fetchMarketLeverageTiers` and `fetchLeverageTiers` are both futures methods, unless bytex has some really unusual max leverage structure for margin trading, but every other exchange I've seen just sets a static max amount for margin trading